src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -17,6 +17,8 @@
 
 package org.apache.commons.io;
 
+import static java.util.Objects.requireNonNull;
```

Comment:
In general, we only use static imports in unit tests for JUnit. These changes creates a lot of clutter in the PR which makes it larger than it needs to be.